### PR TITLE
Instructions

### PR DESCRIPTION
**Pull request by juanjo-ramos**
This PR adds a proper README.md with the instructions to get everyone set up to run the workshop.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,58 @@
...
+We recomnend [Visual Studio Code](https://code.visualstudio.com/). It's also a widely used option at Hudl.
+
+## Running the app
+Congratulations, if you've made it here you have your development environment ready :tada:. Now, you need a place to deploy the fancy animations you'll be creating.
+
+There are 2 options: Running it on the device and on the simulator. Ideally, we could all have them running on the devices. That requires your phone and computer to be on the same network. However, since we don't know the quality of network we'll have on the venue, we highly recommend that you also install one of the simulators as a fallback option in case network isn't good enough.
+
+## Option 1: Run it on your iOS or Android simulator.
+1. Install iOS or Android simulators, if you don't have them already:
+    1. For iOS, [install Xcode](https://itunes.apple.com/es/app/xcode/id497799835?l=en&mt=12).
```
Comment (aichamorro) [inline README.md]: You need to install the simulators too, not just Xcode. It's a huge download, so it's important that the attendands have them downloaded already. Probably the command line tools too..

File: README.md [hunk 2]
```diff
@@ -1,8 +1,58 @@
...
+    2. In the second terminal window, run `yarn start`. That should open a new tab in your browser where expo would start doing it's magic.
+    3. In the tab thas was just opened, click on `Run on Android device / emulator` or `Run on iOS simulator` depending on what you installed in the first step.
+<img width="288" alt="How expo looks like" src="https://user-images.githubusercontent.com/2914389/54987283-9b511280-4fb4-11e9-8cd9-164e08bb0eb4.png">
+After your simulator launches, you should see something like this.
+    ![running expo](https://user-images.githubusercontent.com/2914389/54988086-051dec00-4fb6-11e9-8c6e-6bcb58ef06ed.gif)
+
+You're all set :slightly_smiling_face:.
+
+### Option 2: Run it on your Android or iOS device
+1. Create an account on [expo.io](https://expo.io/)
```
Comment (aichamorro) [inline README.md]: not sure this is required? I don't remember creating an expo account.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,58 @@
 # React Native Animation Workshop
-This project contains all the information to implement the different animations of the React Native Workshop in BitsConf 2019.
+Welcome to the workshop! You will be implementing some animations using React Native. Before you do that, we need to get you set up.
 
-# TODO
-* Add setup instructions
-* Include videos or GIFs of the animations to be implemented
-* Include references to the documentation most relevant for this
-* Include link to the slides so the sample code is available
+We **recommend that you use a MAC** to run the workshop or pair up with someone who has a MAC.
```
Comment (aichamorro) [inline README.md]: are the instructions for windows machines the same as Mac? (apart from the obvious Xcode download).

File: README.md (hunk 2 above)
Comment (juanjo-ramos) [reply README.md]: How does the app on your phone know about your project then? I couldn't find any other option.

If we can remove 1 step it'd be great.

File: README.md (hunk 3 above)
Comment (juanjo-ramos) [reply README.md]: No. I don't even think that you have homebrew for Windows for instance. Should we update all the installation steps to reflect it is Windows only?

File: README.md (hunk 1 above)
Comment (juanjo-ramos) [reply README.md]: I _think_ Xcode comes with some simulators by default, doesn't it? I believe we also download older versions to test Modi in all supported versions but for the workshop I think it'll be fine with what it brings by default.

I'll add a note about command line tools though.

File: README.md (hunk 3 above)
Comment (juanjo-ramos) [reply README.md]: I've added a note at the beginning of the doc for Windows users not to rely on Homebrew but the linked pages for each component.

File: README.md (hunk 1 above)
Comment (aichamorro) [reply README.md]: yeah, that sounds familar

File: README.md (hunk 2 above)
Comment (aichamorro) [reply README.md]: I think the QR code is something like `exp://192.168.1.xxx`

Comment (aichamorro) [reply README.md]: I know you can publish snacks to expo. In that case you do need an account.

Comment (aichamorro) [reply README.md]: I checked my phone and it seems like I am logged into expo, so you might be right here.

**Pull request merged**